source/agora/utils/TxBuilder.d

```diff
@@ -299,7 +299,7 @@ public struct TxBuilder
     public Transaction sign (
         in OutputType outputs_type = OutputType.Payment, uint unlock_age = 0,
         Unlock delegate (in Transaction tx, in OutputRef out_ref) @safe nothrow
-        unlocker = null) @safe nothrow
+        unlocker = null, in Amount fees = Amount(10_000)) @safe nothrow
```

Comment:
Let's make this a member of the struct that can be `set` instead of yet another `sign` parameter.

Reply:
Will make setter for this.